spec/services/healthcheck_component/sengrid_spec.rb

```diff
@@ -2,34 +2,27 @@
 
 RSpec.describe HealthcheckComponent::Sendgrid, type: :service do
 
-  it_behaves_like 'a healthcheck component'
-
   subject { described_class.new }
 
-  context 'sengrid is available' do
-    let(:successful_smtp_response) do
-      instance_double Net::SMTP::Response, success?: true
-    end
+  let(:response_success) { true }
```

Comment:
I like the way the `response_status` has been scooped to the sendgrid is unavailable context, lets move this one into the sendgrid is available context.